hubble/src/main.rs

```diff
@@ -1,7 +1,4 @@
-#![feature(more_qualified_paths)]
-#![feature(try_blocks)]
-#![allow(clippy::needless_lifetimes, clippy::manual_async_fn)]
-#![feature(async_closure)]
```

Comment:
were we not actually using these features?

Reply:
I think we only used them in the original indexers